packages/manager/apps/web-hosting/public/translations/common/Messages_fr_FR.json

```
@@ -73,5 +73,13 @@
   "web_hosting_dashboard_offer_hosting-performance-2": "Performance 2",
   "web_hosting_dashboard_offer_hosting-performance-3": "Performance 3",
   "web_hosting_dashboard_offer_hosting-performance-4": "Performance 4",
-  "web_hosting_dashboard_action_statistics": "Statistiques des visites"
+  "web_hosting_dashboard_action_statistics": "Statistiques des visites",
+  "web_hosting_dashboard_action_import_wordpress_site": "Importer mon site",
+  "web_hosting_dashboard_action_import_wordpress_backlink": "Retour vers mes sites",
+  "web_hosting_dashboard_action_import_wordpress_url_connexion": "Url de connexion",
+  "web_hosting_dashboard_action_import_wordpress_url": "Url d'administration du site",
+  "web_hosting_dashboard_action_import_wordpress_logins_connexion": "Compte administrateur",
+  "web_hosting_dashboard_action_import_wordpress_login": "Compte administrateur",
```

```suggestion
  "web_hosting_dashboard_action_import_wordpress_logins_connexion": "Compte administrateur",
  "web_hosting_dashboard_action_import_wordpress_login": "Compte administrateur",
```
Maybe we can reuse the same for avoid double